src/components/Picture.jsx

```diff
@@ -73,16 +75,23 @@ class Picture extends React.PureComponent {
 }
 
 Picture.propTypes = {
-    sources: PropTypes.array,
+    sources: PropTypes.arrayOf(
+        PropTypes.shape({
+            srcSet: PropTypes.string.isRequired,
+            media: PropTypes.string,
+            type: PropTypes.string.isRequired,
+        })
+    ),
     src: PropTypes.string.isRequired,
-    alt: PropTypes.string,
+    alt: PropTypes.string.isRequired,
     className: PropTypes.string,
     sizes: PropTypes.string,
 };
 
 Picture.defaultProps = {
     src:
         "data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==",
+    alt: "",
```

Comment:
What do you think about removing this so we don't "silence" the proptype warning in case people don't use the `alt` prop? 

I think I prefer that so people are aware of any missing props.

Reply:
Well, that's not to _silence_ `PropType` warnings, this default value is there to guarantee that every image is going to have a default empty `alt` tag and not having it at all.
This is to comply with the [spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#Usage_notes).

Reply:
What I meant was that having this here means that people won't get any warning if they don't set the `alt` prop. I think we should make the warning visible but still render an empty `alt` tag if they ignore the warning.

This can be achieved by removing this from the default props and adding a default `""` in the `renderImage` function above.

What do you think about this solution?

Reply:
@CloudPower97 did you have a chance to look into this? It would be good to know your opinion!

Reply:
@braposo Sure, here we go. Sorry for being so late. :smiley:

Reply:
Not at all! This is open source, every one just helps as much as they can! Please don't feel pressured to do any work 😜 thanks for the changes